WatchApp Extension/Extensions/WKInterfaceImage.swift

```diff
@@ -9,9 +9,12 @@
 import WatchKit
 
 enum LoopImage: String {
-    case fresh
-    case aging
-    case stale
+    case fresh_open
```

Comment:
[nit] it'd be a little cleaner to keep staleness just three cases and add open/closed state as a separate var, rather than exponentially expanding the case count. Would simplify client code using it a bit, too.